node-notes-cli/app.js

```diff
@@ -0,0 +1,45 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+async function parse() {
+  const file = await readFile('data.json');
+  const parse = await JSON.parse(file);
+  return parse;
```

Comment:
No need to define the local `parse` variable. You can just `return await JSON.parse(file);`.